src/components/desktop/desktop-toolbar.vue

```diff
@@ -1,44 +1,53 @@
 <template>
   <div
     v-if="$x.totalResults"
-    class="x-list x-list--horizontal x-list--gap-06 x-list--justify-end x-list--align-center"
+    class="x-list x-list--horizontal x-list--gap-08 x-list--justify-end x-list--align-center"
     data-test="total-results"
   >
-    <span>
-      {{ $t('totalResults.message', { totalResults: $x.totalResults }) }}
-    </span>
+    <span
+      v-html="
+        $t('totalResults.message', { totalResults: $x.totalResults, query: $x.query.searchBox })
+      "
+      class="x-list__item--expand"
+    />
 
-    <ColumnPicker v-slot="{ column }" :columns="columnsValues">
-      <span v-if="column === 0">auto</span>
-      <span v-else>{{ column }}</span>
-    </ColumnPicker>
+    <ColumnPicker />
 
     <BaseIdModalOpen
       modalId="right-aside"
       class="x-button x-button--ghost x-button--ghost-start"
       data-test="toggle-facets-button"
     >
-      <template #default="{ isPanelOpen }">
-        <FiltersIcon />
-        <span v-if="isPanelOpen">{{ $t('toggleAside.hideAside') }}</span>
-        <span v-else>{{ $t('toggleAside.showAside') }}</span>
-      </template>
+      <FiltersIcon />
+      <span>{{ $t('toggleAside.showAside') }}</span>
+      <span
+        v-if="$x.selectedFilters.length"
+        class="
+          x-background--accent
```

Comment:
I think that these classes are ok for the moment, they are not perfect but we will work on the styles later. Maybe we should add what you suggested, a badge class like:

x-badge--in-line or something like that.

Reply:
Yep, with an inline variant of the badge component this would be easier and the resultant styles better.